mmseg/utils/util_distribution.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from mmcv.parallel import MMDataParallel, MMDistributedDataParallel
+
+dp_factory = {'cuda': MMDataParallel, 'cpu': MMDataParallel}
+ddp_factory = {'cuda': MMDistributedDataParallel}
+
+
+def build_dp(model, device='cuda', default_args=None):
+    """build DataParallel module by device type.
+
+    if device is cuda, return a MMDataParallel model; if device is mlu,
+    return a MLUDataParallel model.
+    Args:
+        model(nn.Module): model to be parallelized.
+        device(str): device type, ``'cuda'``, ``'cpu'`` or ``'mlu'``. Defaults to ``'cuda'``.
+        default_args(dict): including the following parameters.
+            device_ids(int): device ids of modules to be scattered to.
+                Defaults to None when GPU or MLU is not available.
+    Returns:
+        model(nn.Module): the model to be parallelized.
```

Comment:
there is not variable in Returns
```suggestion
       nn.Module: the model to be parallelized.
```